flutter/shell/platform/tizen/public/flutter_platform_view.h

```diff
@@ -78,6 +79,7 @@ class PlatformViewFactory {
   virtual PlatformView* Create(int view_id,
                                double width,
                                double height,
+                               double pixel_ratio,
```

Comment:
Shouldn't we pass physical pixel values in `Create`, `Offset`, and `Resize`? Logical sizes are values internally used by the framework (in Dart code), and it's the embedder's responsibility to process the values into raw pixels. See the external texture API (`flutter_texture_registrar.h`) for comparison.

Reply:
I'll remove the pixel_ratio from the argument again.
The reason is that the `platformViewChannel` cannot obtain the required dpi when calculating the pixel_ratio.

> Shouldn't we pass physical pixel values in Create, Offset, and Resize?

It looks good to pass the logical lengths from the dart code to `PlatformView`. `PlatformView` calculates and uses `pixel_ratio`.

Reply:
> The reason is that the `platformViewChannel` cannot obtain the required dpi when calculating the pixel_ratio.

Why? Please, please discuss with your colleagues before making any decision.

> It looks good to pass the logical lengths from the dart code to `PlatformView`.

I have explained the reason for my suggestion. What's your rationale?

Reply:
> Why? Please, please discuss with your colleagues before making any decision.

I'm sorry. I'll discuss it with my colleague before I make a decision next time.
> I have explained the reason for my suggestion. What's your rationale?

we need `pixel_ratio` to convert `logical pixel` to `physical pixel`. By the way, in order to get `pixel_ratio` in flutter-tizen, `evas` or `evas_object` is required. These are variables that we have in `PlatformView(WebView)`, and these are variables that we can't get in `PlatformViewChannel`, so I think we should convert `logical pixel` to `physical pixel` in `PlatformView`.